source/common/config/type_to_endpoint.h

```diff
@@ -10,12 +10,15 @@ namespace Envoy {
 namespace Config {
 
 // Translates an xDS resource type_url to the name of the delta gRPC service that carries it.
-const Protobuf::MethodDescriptor& deltaGrpcMethod(absl::string_view resource_type_url);
+const Protobuf::MethodDescriptor& deltaGrpcMethod(absl::string_view resource_type_url,
+                                                  envoy::config::core::v3::ApiVersion version);
```

Comment:
Please use names like `transport_api_version` for args to make clear which version is which.